app/controllers/miq_policy_controller.rb

```diff
@@ -574,33 +586,40 @@ def replace_right_cell(nodetype, replace_trees = [], presenter = nil)
         if @profiles
           r[:partial => 'profile_list']
         elsif @policies || (@view && @sb[:tree_typ] == 'policies')
-          right_cell_text = _("All %{typ} %{model}") % {:typ => "#{ui_lookup(:model => @sb[:nodeid])} #{@sb[:mode] ? @sb[:mode].capitalize : ""}", :model => ui_lookup(:models => "MiqPolicy")}
+          right_cell_text = _("All %{typ} %{model}") % {
+            :typ   => "#{ui_model_from_id(@sb[:nodeid])} #{@sb[:mode] ? @sb[:mode].capitalize : ""}",
```

Comment:
I find it clearer to have the ui_lookup here instead of a new function wrapping it.
That would also make it self evident what this change is doing.